openvdb_houdini/CMakeLists.txt

```diff
@@ -372,3 +372,8 @@ install(FILES
   DW_OpenVDBRasterizePoints.cmd
   DESTINATION ${OPENVDB_HOUDINI_INSTALL_PREFIX}/scripts/sop
 )
+
+install(FILES
```

Comment:
This should probably be an opt-in mechanism so that it doesn't wipe out an existing pythonrc.py file that already exists in this location. By default I think the base installation directory is $HOUDINI_USER_PREF_DIR which is obviously not specific to VDB.

Reply:
Agreed.